apps/user-office-frontend/src/components/proposalBooking/CreateUpdateVisit.tsx

```diff
@@ -86,6 +86,10 @@ function CreateUpdateVisit({ event, close }: CreateUpdateVisitProps) {
               setFieldValue('team', team);
             }}
             users={values.team || []}
+            pi={undefined}
+            setPi={function (): void {
+              throw new Error('Function not implemented.');
+            }}

```

Comment:
This can be made optional if it not needed by other components  we will not need to pass it to them.